### PR TITLE
Chronos: Remove deprecated API in `MTNet` ut

### PR DESCRIPTION
**Pull request by liangs6212**
related issue: #4095

### CODE REVIEW

**Review by TheaperDeng — COMMENTED** [review]
is it possible to use tsdataset API in this test? If that requires more effort then don't. Current code LGTM.

**Comment by liangs6212** [comment]
> is it possible to use tsdataset API in this test? If that requires more effort then don't. Current code LGTM.

Replace completed.

File: python/chronos/test/bigdl/chronos/forecaster/test_mtnet_forecaster.py [hunk 1]
```diff
@@ -16,57 +16,59 @@
...
-        self.x_test = gen_test_sample(data=np.random.randn(16, 4),
-                                      past_seq_len=look_back)
-
     def test_forecast_mtnet(self):
         # TODO hacking to fix a bug
-        target_dim = 1
-        model = MTNetForecaster(target_dim=target_dim,
+        train_data, data = create_data()
+        self.x_train, y_train = train_data.to_numpy()
+        self.y_train = y_train[:, :, 0].reshape(y_train.shape[0], 1)
```
Comment (TheaperDeng) [inline python/chronos/test/bigdl/chronos/forecaster/test_mtnet_forecaster.py:64]: I think y_train[:, :, 0] has already had shape in (y_train.shape[0], 1), no?

File: python/chronos/test/bigdl/chronos/forecaster/test_mtnet_forecaster.py [hunk 2]
```diff
@@ -16,57 +16,59 @@
...
+        self.x_val, y_val = data.to_numpy()
+        self.y_test = self.y_val = y_val[:, :, 0].reshape(y_val.shape[0], 1)
```
Comment (TheaperDeng) [inline python/chronos/test/bigdl/chronos/forecaster/test_mtnet_forecaster.py:66]: remove `self.y_test = ` here?

File: python/chronos/test/bigdl/chronos/model/test_mtnet.py [hunk 3]
```diff
@@ -19,22 +19,52 @@
...
+    return tsdata_train, tsdata_test
+
+
 class TestMTNetKeras(ZooTestCase):
 
     def setup_method(self, method):
         tf.keras.backend.clear_session()
-        self.ft = TimeSequenceFeatureTransformer()
-        self.create_data()
+        train_data, data = create_data()
```
Comment (TheaperDeng) [inline python/chronos/test/bigdl/chronos/model/test_mtnet.py:59]: same comments here

**Review by TheaperDeng — APPROVED** [review]
LGTM, run jenkins please

**Comment by liangs6212** [comment]
http://10.112.231.51:18888/job/BigDL-Chronos-PR-Validation/407/

**Pull request merged**